qt5/dbusaddons/fcitxqtinputcontextproxy.cpp

```diff
@@ -117,6 +127,11 @@ QDBusPendingReply<> FcitxQtInputContextProxy::invokeAction(unsigned int action,
     return d->icproxy_->InvokeAction(action, cursor);
 }
 
+QDBusPendingReply<bool> FcitxQtInputContextProxy::isVirtualKeyboardVisible() {
+    Q_D(FcitxQtInputContextProxy);
+    return d->icproxy_->IsVirtualKeyboardVisible();
```

Comment:
This is a synchronous dbus call. we should never do that.

Please implement a dbus signal and cache the state in input context instead.